src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -51,17 +53,20 @@ public AdbRunner(ILogger log, IAdbProcessManager processManager, string adbExePa
                 _log.LogDebug($"Using {AdbEnvironmentVariableName} environment variable ({environmentPath}) for ADB path.");
                 adbExePath = environmentPath;
             }
+
             if (string.IsNullOrEmpty(adbExePath))
             {
                 adbExePath = GetCliAdbExePath();
             }
 
             _absoluteAdbExePath = Path.GetFullPath(Environment.ExpandEnvironmentVariables(adbExePath));
+
             if (!File.Exists(_absoluteAdbExePath))
             {
                 _log.LogError($"Unable to find adb.exe");
                 throw new FileNotFoundException($"Could not find adb.exe. Either set it in the environment via {AdbEnvironmentVariableName} or call with valid path (provided:  '{adbExePath}')", adbExePath);
             }
+
             if (!_absoluteAdbExePath.Equals(adbExePath))
             {
                 _log.LogDebug($"ADBRunner using ADB.exe supplied from {adbExePath}");

```

Comment:
this line is already logging the thing this PR purports to be solving?

Reply:
Yeah, but people explicitly asked to get the .exe path only not to rely on parsing some text output.

Also we will need this functionality too in Helix SDK to do some device cleaning so it's just easier to output the path only. Adding a new command seemed like an over-kill